src/Common/src/Interop/Windows/Gdi32/Interop.RegionFlags.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+internal static partial class Interop
+{
+    internal static partial class Gdi32
```

Comment:
I put this top level in WinForms as it's used across dlls.

Reply:
I opted to move it to the global `Interop` class (i.e. Interop.RegionType)

However, there's only like 6 files that are top level in the Interop folder. I thought it may have been a bit much to put this class that is only used in System.Drawing.Common in the root directory
![image](https://user-images.githubusercontent.com/1275900/61860641-335afc00-aec2-11e9-9661-2ec718866070.png)